roles/kubernetes-apps/persistent_volumes/azuredisk-csi/templates/azure-csi-storage-class.yml.j2

```diff
@@ -6,6 +6,9 @@ metadata:
 provisioner: disk.csi.azure.com
 parameters:
   skuname: {{ storage_account_type }}
+{% if azure_csi_tags is defined %}
+  tags: {{ azure_csi_tags }}
```

Comment:
I cannot find why this parameter adds the specified tags to the provisioned volumes.
Could you show that?

Reply:
Documentation: https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/master/docs/driver-parameters.md

Reply:
Seems good, agreed @oomichi?

Reply:
Thanks for your explanation @gutek9 !
That is good for me :-)

/lgtm

Reply:
👍 :-)